packages/amplify_storage_s3/ios/Classes/AmplifyStorageOperations.swift

```diff
@@ -34,101 +36,102 @@ public class AmplifyStorageOperations {
                         result["key"] = key
                         flutterResult(result)
                     case .failure(let storageError):
-                        prepareError(flutterResult: flutterResult, msg: FlutterStorageErrorMessage.UPLOAD_FILE_FAILED.rawValue, localizedError: storageError.errorDescription, recoverySuggestion: storageError.recoverySuggestion)
+                        prepareError(flutterResult: flutterResult, error: storageError as Error)
```

Comment:
Isn't `storageError` already conforms to `Error`?

Reply:
Fixed.